From f02eec93da241cca626f022de87da830d6870dde Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Sun, 27 Jun 2021 19:55:45 +0300
Subject: [PATCH] [SW-46517] greco: change location of rom export fn elf

Since ppboot uses arcboot ROM export function to read from spi, perform
dma and perform verification of image, ppboot compilation requires
export.elf file created out of arcboot ROM compilation. The export.elf
generated with latest arcboot code doesn't match with the arcboot ROM
flashed on greco cards. In order to resolve issue, export.elf is created
with the arcboot ROM code which is available on greco card and
copied to buildroot external folder. This patch add changes to use
pre compiled arboot rom export elf.

Change-Id: Iddc2eb1b78379a6445d9de6d1f521053d83548b8
---

diff --git a/Makefile b/Makefile
index 0461105..3b85970 100644
--- a/Makefile
+++ b/Makefile
@@ -61,7 +61,7 @@
 # compile binaries based on target
 ifeq ($(HL_DEV),$(filter $(HL_DEV),goya2_arc greco_arc))
 	BINARIES = $(PPBOOT_TARGET) $(PCI_PHY_FW_TARGET)
-	LDFLAGS += -Wl,--just-symbols ../arcboot-custom/exports.elf
+	LDFLAGS += -Wl,--just-symbols ${BR2_EXTERNAL}/board/habanalabs/$(HL_DEV)/exports.elf
 else ifeq ($(HL_DEV),$(filter $(HL_DEV),goya2_arc_gnu greco_arc_gnu))
 	BINARIES = $(PREBOOT_TARGET)
 else
